ds-controller-board/device/config.go

```diff
@@ -4,15 +4,38 @@
 package device
 
 import (
-    "time"
+	"time"
 )
 
+// CustomConfig holds the values for the driver configuration
+type CustomConfig struct {
+	DriverConfig Config
+}
+
+// ServiceConfig a struct that wraps CustomConfig which holds the values for driver configuration
+type ServiceConfig struct {
+	AppCustom CustomConfig
+}
+
 // Config is the global device configuration, which is populated by values in
 // the "Driver" section of res/configuration.toml
 type Config struct {
-    VirtualControllerBoard bool
-    PID                    string
-    VID                    string
-    DisplayTimeout         time.Duration
-    LockTimeout            time.Duration
+	VirtualControllerBoard bool
+	PID                    string
+	VID                    string
+	DisplayTimeout         time.Duration
+	LockTimeout            time.Duration
```

Comment:
So will the TOML marshaler properly marshal the strings into these  time.Duration fields?
Please try putting invalid duration values to see how it handles it.